test/functional/page_objects/timelion_page.js

```diff
@@ -59,12 +60,13 @@ export function TimelionPageProvider({ getService, getPageObjects }) {
       return await Promise.all(elements.map(async element => await element.getVisibleText()));
     }
 
-    async clickSuggestion(suggestionIndex = 0) {
+    async clickSuggestion(suggestionIndex = 0, waitTime = 500) {
       const elements = await find.allByCssSelector('.suggestions .suggestion');
       if (suggestionIndex > elements.length) {
         throw new Error(`Unable to select suggestion ${suggestionIndex}, only ${elements.length} suggestions available.`);
       }
       await elements[suggestionIndex].click();
+      await PageObjects.common.sleep(waitTime);
```

Comment:
thoughts on adding a small comment fo why we have a sleep?

Reply:
Thanks @jdbudz The sleeps were in the original test I just moved them from the test to the page object.  In the original test the sleeps were placed after clickSuggestion() method, but I noticed in the original not every call to clickSuggestion() had a sleep, so it could cause flakiness.   Moving them to the page object ensures a sleep after click.  The sleep is needed since there is no event indication like page loading to determine when the timeline expression is updated after clicking a timelion suggestion.